users/src/main/kotlin/com/eros/users/models/UserEnums.kt

```
@@ -681,3 +689,49 @@ enum class MediaType {
     PHOTO,
     VIDEO
 }
+
+/**
+ * Reach Level - for how flexible a user's preferences are used for matching.
+ */
+enum class ReachLevel() {
```

detekt (1.23.8)

[warning] An empty default constructor can be removed.

(detekt.empty-blocks.EmptyDefaultConstructor)